Add remote execution permission (#1304)
theforeman/foreman-documentation: guides/common/modules/ref_predefined-roles.adoc

| Edit partition tables | View, create, edit and destroy partition tables.
| Manager | A role similar to administrator, but does not have permissions to edit global settings.
In the {ProjectWebUI}, global settings can be found under *Administer* > *Settings*.
This role is one of two with the ability to grant the `execute_jobs_on_infrastructure_hosts` permission.
| Organization admin | An administrator role defined per organization.
The role has no visibility into resources in other organizations.
| Red{nbsp}Hat Access Logs | View the log viewer and the logs.

> | Remote Execution Manager | A role with full remote execution permissions, excluding {ProjectName} and {Capsules} hosts, and including modifying job templates.

Reviewer: ```suggestion
| Remote Execution Manager | A role with full remote execution permissions, excluding {ProjectServer} and {SmartProxies}, and including modifying job templates.
```
https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L93

> | Remote Execution User | Run remote execution jobs against hosts, excluding {ProjectName} and {Capsules}.

Reviewer: same as above.

Reviewer 2: The voice and mood changes in this sentence and breaks parallelism with the other examples.

Author: @melcorr , the info in this was suggested by the developers to highlight the new REX capabilities.  Are you recommending I take out the parts like "excluding XYZ hosts and including modifying..." to make it less specific?

Reviewer 2: Information comes from developers. The writing style is the remit of the writer. 
What I am saying is to follow our style with regards to parallelism.

> | Site manager | A restrained version of the Manager role; this role has the ability to grant the `execute_jobs_on_infrastructure_hosts` permission.

Reviewer: ```suggestion
| Site manager | A restrained version of the Manager role. This role can grant the `execute_jobs_on_infrastructure_hosts` permission to other users.
```

| Organization admin | An administrator role defined per organization.
The role has no visibility into resources in other organizations.
| Red{nbsp}Hat Access Logs | View the log viewer and the logs.
| Remote Execution Manager | Control which roles have permission to run infrastructure jobs
| Remote Execution User | Run remote execution jobs against hosts

> | Site manager | A restrained version of the Manager role. 

Reviewer: ```suggestion
| Site manager | A restrained version of the Manager role.
```

> | Remote Execution User | Run remote execution jobs against hosts, excluding {ProjectServers} and {SmartProxies}.

Reviewer 2: This is the sentence that breaks parallelism with the rest of your entries here that I flagged in my previous review.
